tests/Scripts/Configure-FunctionalTests.ps1

```diff
@@ -0,0 +1,43 @@
+[CmdletBinding(DefaultParameterSetName='RegularBuild')]
+param (
+    [string]$ServiceRoot,
+    [string]$Slot,
+    [string]$CloudServiceName,
+    [string]$SubscriptionId,
+    [string]$ApplicationId,
+    [string]$TenantId,
+    [string]$AzureCertificateThumbprint
+)
+
+# Determine the url to run the tests against
+if ($Slot -eq "Production")
+{
+    # If the slot is 'Production', then we can assume that we are testing the same url as the service root. 
+    $SearchServiceUrl = $ServiceRoot
+}
+else
+{
+    # Use Azure PowerShell cmdlets to find the url of the desired slot
+    Try
+    {
+        Write-Host "Logging into Azure as service principal."
+        # Log into Azure using a service principal with a certificate
+        $login = Add-AzureRmAccount -ApplicationId "$ApplicationId" -CertificateThumbprint "$AzureCertificateThumbprint" -ServicePrincipal -SubscriptionId "$SubscriptionId" -TenantId "$TenantId"
```

Comment:
Good catch...I think at some point I used it but no longer needed to. The function still needs to be called though, but `$login` does not to be saved.